src/github/copilotRemoteAgent.ts

```diff
@@ -306,8 +306,8 @@ export class CopilotRemoteAgentManager extends Disposable {
 		};
 
 		let autoPushAndCommit = false;
-		const message = vscode.l10n.t('Copilot coding agent will continue your work in \'{0}\'.', repoName);
-		const detail = vscode.l10n.t('Your current chat session will end, and its context will be used to continue your work in a new pull request.');
+		const message = vscode.l10n.t('Coding agent will continue your work in \'{0}\'.', repoName);
```

Comment:
I think this (and the instance below) should be `The coding agent`, but open to feedback